skrf/media/media.py

```diff
@@ -646,11 +649,9 @@ def impedance_mismatch(self, z1: NumberLike, z2: NumberLike, **kwargs) -> Networ
         resistor
         """
         result = self.match(nports=2, **kwargs)
+        s_def = kwargs.get('s_def', S_DEF_DEFAULT)
         gamma = tf.zl_2_Gamma0(z1,z2)
```

Comment:
```suggestion
        gamma = tf.zl_2_Gamma0(z1,z2)
```
gamma is no more useful